scripts/requirements-doc.txt

```diff
@@ -5,6 +5,7 @@ sphinx~=3.3
 sphinx_rtd_theme>=0.5.2,<1.0
 sphinx-tabs
 sphinxcontrib-svg2pdfconverter
+pygments~=2.9
```

Comment:
>=?

Reply:
~= is the compatible version specifier, means `>=2.9,<3`. I can't find if pygments uses semver though...

Reply:
OK, fair